cookbooks/bcpc-hadoop/templates/default/hv_hive-site.xml.erb

```diff
@@ -166,7 +166,7 @@
 <property>
   <name>hive.stats.dbconnectionstring</name>
   <value>jdbc:mysql:loadbalance://<%=@mysql_hosts.join(',')%>:3306/hive_table_stats?useUnicode=true&amp;
-characterEncoding=UTF-8&amp;user=<%=get_config("mysql-hive-table-stats-user")%>&amp;password=<%=get_config("mysql-hive-table-stats-password")%></value>
+characterEncoding=UTF-8&amp;user=<%=get_config("mysql-hive-table-stats-user")%>&amp;password=<%=get_config("password", "mysql-hive-table-stats", "os")%></value>
```

Comment:
This should not put a bare password in the configuration file but instead use the Java keystore as Chef-BACH #266 does.